blues/ncmc.py

```diff
@@ -345,27 +348,30 @@ def create_normalSystem(self, coord_file, top_file):
 #        self.dummy_integrator = openmm.openmm.LangevinIntegrator(temperature, friction, timestep)
 #        self.md_simulation = openmm.app.simulation.Simulation(topology=self.normalsystem.topology, system=self.normalsystem.system, integrator=self.md_integrator)
 #        self.dummy_simulation = openmm.app.simulation.Simulation(topology=self.normalsystem.topology, system=self.normalsystem.system, integrator=self.dummy_integrator)
-   
-    def rotationalMove(self, residueList=None):
+
+    def rotationalMove(self, context=None, residueList=None,):
         """
         Function to be used in movekey. Performs a rotation around the center of mass
         of the ligand.
         Arguments
         ---------
         generally none
```

Comment:
This takes an optional `context` argument now, so it should be explained what that does. Also, what happens if I give a residue list?